### PR TITLE
RELATED: BB-2381 drill to url

### PR DESCRIPTION
**Pull request by yanlipnican**

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R208](https://confluence.intgdc.com/display/Dev/R208).
This release is **zero downtime** release.
Code drop: 2020-07-02 00:00.

**Comment by yenkins** [comment]
Story [BB-2381](http://jira.intgdc.com/browse/BB-2381) Allow to post message with URL instead of redirecting the frame in embedded mode
	Status: Hacking
	Priority: Blocker
	Assignee: Jan Lipnican

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/117/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/201/ : FAILURE in 59s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/118/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/202/ : SUCCESS in 1m 13s

File: src/embedding/kd.ts [hunk 1]
```diff
@@ -186,7 +186,20 @@ export namespace EmbeddedKpiDashboard {
         /**
          * Type represent that the remove filter context action is finished
          */
-        RemoveFilterContextFinished = 'removeFilterContextFinished'
+        RemoveFilterContextFinished = 'removeFilterContextFinished',
+
+        /**
+         * Type that represents the drill to url was started and loading can be displayed. It does not contain url.
+         * Url is contained by DrillToUrlResolved which is posted after url is resolved. Event also contains ID which
```
Comment (xMort) [inline src/embedding/kd.ts:193]: I see some grammar errors. Maybe rephrase it like this (or in similar manner, maybe my text is not without errors as well).

"Type that represents started drill to URL. The event does not contain an URL. The event can be used as notification to display a loading indicator as the URL resolving takes some time. 
The URL is sent in DrillToUrlResolved event which is posted after the URL is resolved. The event also contains an ID that can be matched with ID in subsequently posted DrillToUrlResolved event."

File: src/embedding/kd.ts [hunk 2]
```diff
@@ -186,7 +186,20 @@ export namespace EmbeddedKpiDashboard {
...
+         * can be then matched with DrillToUrlResolved ID
+         */
+        DrillToUrlStarted = 'drillToUrlStarted',
+
+        /**
+         * Type that represents the drill to url was resolved after DrillToUrlStarted and contains resolved url. Event
+         * also contains ID which can be matched with DrillToUrlStarted ID
+         */
```
Comment (xMort) [inline src/embedding/kd.ts:201]: Same here:

"Type that represents resolved drill to URL. The event is sent after DrillToUrlStarted event was posted and it contains the resolved URL. The event also contains an ID which can be matched with ID from DrillToUrlStarted event."

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/119/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/203/ : SUCCESS in 2m 34s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/42/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/204/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-fossa-scanning-zuul-el7/6/ : SUCCESS in 13s (non-voting)
